app/assets/javascripts/vendor/lib/google/maps.js.erb

```diff
@@ -1,419 +1,417 @@
+      //if (sw['lng']() > 0 && ne['lng']() < 0) {     // are we spanning the prime meridian
+      //  coordList.push([-180.0, sw['lat']()]);       // point at prime meridian
+      //}
+      coordList.push([center_lat_long['lng'](), sw['lat']()]);       // point at center ALWAYS vs conditional at 0
+      coordList.push([sw['lng'](), sw['lat']()]);
+      var temparray = [];
+      temparray[0] = coordList;
+      coordList = temparray;        // this is an expedient kludge to get [[[lng,lat],...]]
+      var bounds_box = {
+        "type": "Feature",
+        "geometry": {
+          "type": "multilinestring",
+          "coordinates": coordList
+        },
+        "properties": {}
+      };
+      map.data.addGeoJson(bounds_box);
+      return map;             // now no global map object, use this object to add listeners to THIS map
+    },
+
+    doFeatureCollection: function (featureCollection, otu_id, chained) {
+      if (featureCollection.features.length > 0) {
+        for (var i = 0; i < featureCollection.features.length; i++) { // this loop looks for (currently) checkboxes that
+          var this_feature = featureCollection.features[i];           // indicate inclusion into the google maps features
+          if (this_feature.properties) {
+            var this_property_key = this_feature.properties.source_type;
+            var this_control = 'check_' + this_property_key;    // + '_' + OTU_ID
+            if (otu_id != null) {
+              this_control += '_' + otu_id
+            }
+          }
+          if (document.getElementById(this_control) != undefined) {   // if checkbox control exists
+            if (document.getElementById(this_control).checked) {      // if checked, and only
+              this_feature.properties.otu_id = otu_id;
+              chained.features.push(this_feature);                    // if checked, insert this feature/properties
+            }                                                         // otherwise skip this feature
+          }
+          else {                                    // if no corresponding control property, do not block insertion
+            chained.features.push(this_feature);    // functionality for non-checkbox-connected data
+          }
+        }
       }
-      if ((xminp == 0.0) && (xmaxp == 0.0) && (xmaxm == 0.0) && (xminm == 0.0)) {   // no data (kinda)
-        xminm = -180.0;     // this calculation is to reflect
-        xmaxp = 180.0;      // latitude calculation below
+    },
+
+    log_2: function (x) {
+      return Math.log(x) / Math.LN2;  // log_2(x) = ln(x)/ln(2)
+    },
+
+    get_window_center: function (bounds) {      // for use with home-brew geoJSON scanner/enumerator
+      var xminp = bounds.xminp;           //
+      var xmaxp = bounds.xmaxp;           //
+      var xminm = bounds.xminm;           //
+      var xmaxm = bounds.xmaxm;           // these elements already set
+      //if (bounds.prime /*&& bounds.anti*/) {
```

Comment:
This is related to the deferred back-integration of anti-meridian handling into the maps code.